test/expected/drop_chunks.out

```diff
@@ -54,10 +54,11 @@ WHERE h.schema_name = 'public' AND (h.table_name = 'drop_chunk_test1' OR h.table
 (0 rows)
 
 \dt "_timescaledb_internal".*
-      List of relations
- Schema | Name | Type | Owner 
---------+------+------+-------
-(0 rows)
+                     List of relations
```

Comment:
should we filter the`bgw_job_stat` output in this test? It is not related to the test as we only want to see chunks.

Reply:
fixed